float32[6] position_x
float32[6] position_y
bool[6] lidar_update